### DISCUSSION TITLE
nerdctl push jfrog (404)

### DISCUSSION THREAD

**Discussion by rodrigorootrj2** [containerd/nerdctl#794] [General]
Hi dears,
I have head push my container in docker hub, this work. But when i push in jfrog system,my internal hub, the same image not work. Early, i push images in jfrog with docker, the migrate not work for that.
There anything diference betwen push, in jfrog enveroment?
error:
`FATA[0000] failed commit on ref "config-sha256:87116068d67336b34dbb31e5e32922e14aafecf1bf8264116e9d998a9e9a304a": unexpected status: 404 Not Found`

**Comment by AkihiroSuda** [comment]
Could you try v0.16.1 if your version is older?

**Comment by AkihiroSuda** [comment]
(FYI: I renamed the OP title: ` nerdctl push jfrog` -> `nerdctl push jfrog (404)`)

**Comment by fahedouch** [comment]
hello @rodrigorootrj2 ,

I need more element to debug this:

Are you pushing to `local` or `virtual` jfrog repo ?
Which is the docker API version of you jfrog repo ? is it `v2` ? 
which is your jfrog server version ? 

FYI : I tired a multi-platform image and single platform push with `v0.16.1` and it was ok :) ! 
thkx
